scripts/generate-bindings.sh

```diff
@@ -17,8 +17,8 @@ compile-contract() {
 
 	echo "Generating go bindings for ${solidity_type_name}"
 
-	"${SOLC_BIN}" --abi "ethereum/contracts/${solidity_type_name}.sol" -o ethereum/abi/ --overwrite
-	"${SOLC_BIN}" --bin "ethereum/contracts/${solidity_type_name}.sol" -o ethereum/bin/ --overwrite
+	"${SOLC_BIN}" --optimize --optimize-runs=200 --abi "ethereum/contracts/${solidity_type_name}.sol" -o ethereum/abi/ --overwrite
+	"${SOLC_BIN}" --optimize --optimize-runs=200 --bin "ethereum/contracts/${solidity_type_name}.sol" -o ethereum/bin/ --overwrite
```

Comment:
I think 200 runs is already the default.  Did you play with the other optimization flags?
https://docs.soliditylang.org/en/v0.8.19/internals/optimizer.html

Reply:
yeah I tried a bunch of different values (1 - 1000000), this value seemed the best in terms of bytecode size/call gas tradeoffs. increasing the number only increased the bytecode size, with negligible effects on gas. smaller numbers had slightly smaller bytecode but weren't as effective on saving gas